packages/metro/src/ModuleGraph/worker/collectDependencies.js

```diff
@@ -37,6 +37,18 @@ export type Dependency<TSplitCondition> = $ReadOnly<{
   name: string,
 }>;
 
+// TODO: Convert to a Flow enum
+type ContextMode = 'sync' | 'eager' | 'lazy' | 'lazy-once';
+
+type RequireContextParams = $ReadOnly<{
```

Comment:
Seems like we have this definition both here and in `types.flow.js` - we probably need just one of the two?